sympy/matrices/dense.py

```diff
@@ -290,6 +305,126 @@ def rot_axis1(theta):
     return Matrix(lil)
 
 
+def rot_ccw_axis3(theta):
+    """Returns a rotation matrix for a rotation of theta (in radians)
+    about the 3-axis.
+
+    For a right-handed coordinate system, this corresponds to a
+    counterclockwise rotation.
+
+    Examples
+    ========
+
+    >>> from sympy import pi, rot_ccw_axis3
+
+    A rotation of pi/3 (60 degrees):
+
+    >>> theta = pi/3
+    >>> rot_ccw_axis3(theta)
+    Matrix([
+    [      1/2, -sqrt(3)/2, 0],
+    [sqrt(3)/2,        1/2, 0],
+    [        0,          0, 1]])
+
+    If we rotate by pi/2 (90 degrees):
+
+    >>> rot_ccw_axis3(pi/2)
+    Matrix([
+    [0, -1, 0],
+    [1,  0, 0],
+    [0,  0, 1]])
+
+    See Also
+    ========
+
+    rot_ccw_axis1: Returns a rotation matrix for a rotation of theta (in radians)
+        about the 1-axis
+    rot_ccw_axis2: Returns a rotation matrix for a rotation of theta (in radians)
+        about the 2-axis
```

Comment:
It would be good to have "see also" references linking e.g. `rot_axis1` and `rot_ccw_axis1`.

Reply:
Alright, I'll make another PR another day for this.

Is 4 references too much? I could also reference the Givens rotation formula when I make another PR for that.

Reply:
I don't think 4 references is too many. These are definitely relevant references.

In general we should think about references as being like a web. It's less important that each thing references each other thing than that a reference leads to a reference which leads to the other thing. Making 3 functions that reference only each other and 3 very similar functions that reference only each other with no cross references breaks that web.